shell/assets/styles/base/_mixins.scss

```diff
@@ -158,5 +158,6 @@
 
 @mixin focus-outline {
   // Focus for form like elements (not to be confused with basic :focus style)
-  outline: 2px solid var(--primary-keyboard-focus);
+  // we need to use !important because it needs to superseed other classes that might impact outlines
+  outline: 2px solid var(--primary-keyboard-focus) !important;
```

Comment:
We shouldn't be using `!important` if at all possible. Is this the same change from a different PR?

Reply:
it needs rebase. Done